### ISSUE TITLE
can't handle : in input path

### ISSUE THREAD

**Issue by gijzelaerr** [DataBiosphere/toil#1958]
It is interpreted as a separator by the docker engine:
```
docker: Error response from daemon: invalid bind mount spec "/vault-tina/gijs/prefactor-cwl/runs/run_2017-11-14-T14:19:29/results/out_tmpdiriJOjgW:/var/spool/cwl:rw": invalid volume specification: '/vault-tina/gijs/prefactor-cwl/runs/run_2017-11-14-T14:19:29/results/out_tmpdiriJOjgW:/var/spool/cwl:rw'
```

┆Issue is synchronized with this [Jira Story](https://ucsc-cgl.atlassian.net/browse/TOIL-216)
┆Issue Number: TOIL-216

**Comment by mr-c** [comment]
The fix for this requires Docker 17.05 or later: https://github.com/moby/moby/issues/8604#issuecomment-332685815

**Comment by cricketsloan** [comment]
➤ Adam Novak commented:

This should be fixed now that we go through Docker’s API to set up our Docker executions.
